delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/client/QueryBuilder.scala

```diff
@@ -69,6 +69,10 @@ final case class QueryBuilder private[client] (private val query: JsonObject) {
     */
   def withFilters(params: ResourcesSearchParams)(implicit baseUri: BaseUri): QueryBuilder = {
     val (includeTypes, excludeTypes) = params.types.partition(_.include)
+    val value1 = params.fileUserMetadata.toList.flatMap(_.keywords).map { case (key, value) =>
```

Comment:
Can we address the change in the listing operation in a later PR and just make sure that data is properly indexed and full text search on keywords is working ?

Reply:
I can't have a working test without the listing so I think it's needed. We can not do name/description if we want to slice it. Full text search could also be cut given that it's more advanced functionality and isn't mentioned in the ticket

Reply:
The data can be indexed and returned by the listing operation without having an extra param to filter for the moment.

Reply:
I would prefer to implement a full slice of functionality, plus: it's already done